data/governments.txt

```diff
@@ -809,6 +809,8 @@ government "Remnant"
 		capture 10
 		destroy 10
 		scan 0.1
+	"use foreign penalties for"
+		"Remnant (Research)"
```

Comment:
Can we expand this feature to have this be able to be on a per-action basis? For example, let's say it was *only* the scan penalty that you wanted to be different, then you might list something like:
```
"Remnant (Research)" "scan"
```
If no specific penalty action were listed, then all action penalties would be used.

Alternatively, instead of just inheriting the penalties from the other government, we could define custom penalties on a per-government-per-action basis.
```
"use foreign penalties for"
	"Remnant (Research)"

"use custom penalties for" "Remnant (Research)"
	assist -0.25
	disable 1
	board 1
	capture 10
	destroy 10
```

Since this new syntax could be directly expanded from or entirely separate from the old, this can be a future change in a separate PR.

Reply:
I think that custom value per action makes the most sense and I'll allow a "foreign" keyword instead of a number to use the penalty of the other government